learning/tour-of-beam/learning-content/windowing/fixed-time-window/description.md

```diff
@@ -0,0 +1,68 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+### Fixed time windows
+
+The simplest form of windowing is using fixed time windows: given a timestamped `PCollection` which might be continuously updating, each window might capture (for example) all elements with timestamps that fall into a 30-second interval.
+
+A fixed time window represents a consistent duration, non overlapping time interval in the data stream. Consider windows with a 30-second duration: all the elements in your unbounded PCollection with timestamp values from 0:00:00 up to (but not including) 0:00:30 belong to the first window, elements with timestamp values from 0:00:30 up to (but not including) 0:01:00 belong to the second window, and so on.
+
+{{if (eq .Sdk "go")}}
+```
+fixedWindowedItems := beam.WindowInto(s,
+	window.NewFixedWindows(30*time.Second),
+	items)
+```
+{{end}}
+
+{{if (eq .Sdk "java")}}
+```
+PCollection<String> items = ...;
+    PCollection<String> fixedWindowedItems = items.apply(
+        Window.<String>into(FixedWindows.of(Duration.standardSeconds(30))));
+```
+{{end}}
+
+{{if (eq .Sdk "python")}}
+```
+from apache_beam import window
+fixed_windowed_items = (
+    items | 'window' >> beam.WindowInto(window.FixedWindows(30)))
+```
+{{end}}
+
+### Playground exercise 
+
```

Comment:
Please add the description of what a runnable example does out of the box. Such as:

In the playground window, you can try an example of how to create a fixed-time window and print elements in it.